oasis/notebooks.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+description: >-
+  Colocated Jupyter Notebooks within the Oasis platform with full GPU support
+---
+# Notebooks
+
+{{ page.description }}
+{:class="lead"}
+
+ArangoDB Oasis provides an easy way to run Jupyter Notebooks in the cloud with
+full GPU support.
```

Comment:
not sure if we want to add this already, we release notebook [BETA] support to the platform now, which will have no GPU support, this will added before we mark the feature as GA.